api-reference/direct-connects/hotel-service-2/Error-handling.markdown

```diff
@@ -64,17 +64,17 @@ Concur supports the following Error Type Codes in any of the responses:
 
 Concur expects the following Errors under the given Error Types:
 
-### Unknown	(Error Type Code "1")								Indicates an unknown error.
+### Unknown	Error Type Code 1
```

Comment:
You can fix the rendering error by adding a newline between the header and the table.

Reply:
duh!  I've been staring at these for too long and totally missed that.  Thanks!